workflows/sars-cov-2-variant-calling/sars-cov-2-ont-artic-variant-calling/ont-artic-variation.ga

```diff
@@ -103,7 +103,7 @@
                 "y": 710.5
             },
             "tool_id": null,
-            "tool_state": "{\"default\": 700, \"parameter_type\": \"integer\", \"optional\": true}",
+            "tool_state": "{\"default\": 2000, \"parameter_type\": \"integer\", \"optional\": true}",
```

Comment:
I think if there is some benefit in doing this we should probably make this a workflow parameter ?

Reply:
i'm fine with a workflow parameter, but I think it's good to also set the default based on this data (or other generate more data?)